plugin/utils/shell.py

```diff
@@ -86,6 +87,183 @@ def create_lods():
 	return msgs
 
 
+def generate_rig_edit():
+    """Automatic rig edit generator by NDP. Detects posed bones and automatically generates nodes and offsets them."""
+    msgs = []
+    #logging.info(f"Generating rig edit")
+    print("-------------------------------------------------------------")
```

Comment:
Convert print to logging.info or appropriate log level

Reply:
Converted all prints to logging.info